src/Shared/Const/Message.cs

```diff
@@ -71,4 +71,12 @@ public static class AddonMessage
 		/// </summary>
 		public const string KEYBOARD_TUTORIAL = "KEYBOARD_TUTORIAL";
 	}
+
+	/// <summary>
+	/// Constant strings sent with `BC_NORMAL_Run`.
+	/// </summary>
+	public static class BarrackMessage
+	{
+		public const string THEMA_BUY_SUCCESS = "THEMA_BUY_SUCCESS";
```

Comment:
@exectails: Once you have decided whether we should change the styling of these messages, we can change all of them at once.

Reply:
Yea, I'm not sure. What do you think? Seeing upper case underscore naming makes me cringe, but it makes sense from the perspective of someone who digs around the client^^

Reply:
I think it's fine to rename them. Not everything needs to be the same as the client.